First&LastpositionofanArray.java

```diff
@@ -0,0 +1,44 @@
+//Runtime=1ms
+//memory usage:46MB
+//Time complexity O(N) N is the number of elements in an array
+//sent the arr to -1,-1 by default.
+//found the mid element when low is less than or equal to high
+//if mid value less than target then shift low to mid+1
+//if mid value greater than target then shift high to mid-1
+//if target equal to mid value then check low value and if it is less low++
+//if target equal to mid value then check high value and if it is more high--
+//if both nums[low] and nums[high] values are equal to target  then push low and high into the array and return it
+//if low > high . it comes out of while loop and returns -1,-1
+
```

Comment:
Please try to add comments along with the code. It will be easy to understand code.